media/css/firefox/new.less

```diff
@@ -341,7 +341,8 @@ html[lang|="en"] #main-headline span:first-child {
     width: 100%;
 }
 
-.download-button-wrapper.android {
+.download-button-wrapper.android,
+.download-button-wrapper.ios {
```

Comment:
I think we'll need some CSS to display this `.download-button-wrapper.ios` for users *on* iOS. Something like [we do for Android](https://github.com/mozilla/bedrock/blob/master/media/css/firefox/new.less#L274) maybe?